content-usable/index.html

```diff
@@ -1365,27 +1384,21 @@ <h1>Appendix: Mapping User Needs, Persona and Patterns</h1>
-	    </p>
+<p>Many agencies and services are required to use plain language and to be usable by vulnerable groups. This document will help content developers know what to do to achieve this 
+goal across different geographical areas and include user groups of people with learning and cognitive disabilities. In addition many sites want to reach user groups such as millennials 
+with learning disabilities and people with age-appropriate forgetfulness. This can be because of their commitment to inclusion, or to enable growth in these high value, under-serviced, 
+markets. Typically, there are many more people in the target audience with a cognitive or learning disability than the content provider is aware of, and many 
+content providers are often losing these user groups.	    
+</p>
 
-<p>This document is not normative. It is intended to address accessibility barriers that currently not included in the normative WCAG 2.x specification and may not otherwise be met.  Some of the advice in this document may not applicable to every web site. (In contrast conformance to <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">The Web Content Accessibility Guidelines (WCAG)</a> is required by law in many countries, and is designed to be widely applicable to and allow conformance by all web content.) </p>
-<p>User considerations should be taken into account when deciding how to apply this document. For instance, Web content and applications that affect individual safety concerns, health, critical services, autonomy, care-giving, social integration, and workplace needs should follow as much of the advice in this document as possible.</p>
-<p>This section provides guidance and considerations on how to use this document and the design patterns (general, repeatable solutions to commonly occurring problems) to build a policy or requirements regarding web content to ensure that the needs of individuals with learning or cognitive disabilities are addressed. Web content designed without consideration for the needs of individuals with learning or cognitive disabilities may create accessibility barriers to the needs of the end-user. Development of a plan or policy includes the following steps which are discussed in this section:</p>
+<p>User considerations should be taken into account when deciding how to apply this document. For instance, Web content and applications that affect individual safety concerns, 
```

Comment:
I thought we deleted this sentence about user conciderations? Also appears again below. I find it a bit vague anyway.

Reply:
Rachael replied is all correct as is.